config/initializers/version_check.rb

```diff
@@ -0,0 +1,4 @@
+rh_domain = MiqAeDomain.find_by(:name => "RedHat")
```

Comment:
Sorry, but hitting the db in an initializer will cause rake tasks that shouldn't need a db to need one... such as asset compilation.  😢

Reply:
I would suggest we check the version before seeding the datastore:
https://github.com/ManageIQ/manageiq-automation_engine/blob/a13df793acd0096db25182a2e6750f5b6aedde42/app/models/miq_ae_datastore.rb#L214